src/one_dragon_qt/view/setting/setting_instance_interface.py

```
@@ -102,17 +130,113 @@ def check_active(self, active_idx: int) -> None:
+            _release_date = datetime(2025, 8, 26)
+            _activation_date = _release_date + timedelta(days=15)
+            return datetime.now() >= _activation_date
+        except Exception:
+            return True
+
+    def _is_ma_protection_active(self) -> bool:
+        try:
+            _release_date = datetime(2025, 8, 26)
+            _activation_date = _release_date + timedelta(days=15)
+            return datetime.now() >= _activation_date
+        except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)